cypress/integration/galleryDom.spec.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+describe(('Gallery application'), () => {
+  const url = 'http://localhost:8080';
+  const imageSec = '/images/landscape-second.png';
+  const imageThird = '/images/landscape-third.png';
+  const imageFourth = '/images/landscape-fourth.png';
+  const imageFifth = '/images/landscape-fifth.png';
+  const imageFirst = '/images/landscape-first.png';
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.get('[id="largeImg"]').as('largeImg');
+  });
+
+  it('can click on a small `img` image', () => {
+    cy.get('ul li:nth-child(2)').click();
```

Comment:
is there a better selector then "ul li:nth-child(n)"?

Reply:
all li elements are equal, li:nth-child is shortest